addons/l10n_es_edi_verifactu_pos/static/src/overrides/components/reprint_receipt_button/reprint_receipt_button.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+import { OrderReceipt } from "@point_of_sale/app/screens/receipt_screen/receipt/order_receipt";
+import { patch } from "@web/core/utils/patch";
+import { ReprintReceiptButton } from "@point_of_sale/app/screens/ticket_screen/reprint_receipt_button/reprint_receipt_button";
+
+patch(ReprintReceiptButton.prototype, {
+    //@override
+    async click() {
+        if (!this.props.order) {
+            return;
+        }
+        let order = this.props.order;
+        const [res] = await this.pos.selectedOrder.fetch_l10n_es_edi_verifactu_qr_code([order.server_id]);
+        order.l10n_es_edi_verifactu_qr_code = res.l10n_es_edi_verifactu_qr_code;
+
+        (await this.printer.print(OrderReceipt, {
+            data: order.export_for_printing(),
+            formatCurrency: this.env.utils.formatCurrency,
+        })) || this.pos.showScreen("ReprintReceiptScreen", { order: this.props.order });
```

Comment:
```suggestion
       return super.click();
```
Using the super call is preferred here to avoid unnecessary duplication of logic.